sdks/java/nexmark/build.gradle

```diff
@@ -46,3 +46,14 @@ dependencies {
 test {
   jvmArgs "-da"
 }
+
+def nexmarkArgsProperty = "nexmark.args"
+
+task run (type: JavaExec) {
```

Comment:
Is there a more meaningful name you could provide instead of `run`?

nit: `run (` -> `run(` to follow the same style used in all the other build files and Java

Reply:
Well, the reason I chose it is because `run` because Nexmark is an application with one specific entry point. It is the same task that that `application` plugin uses, and we are doing the same thing here. I decided that plugin might not be the right fit, but if you feel differently I'll try it.

Reply:
I was under the impression that each query was its own executable.

Reply:
yes you just run Main